src/components/Layout/UserInfo.tsx

```diff
@@ -58,17 +61,30 @@ const UserInfo = () => {
     <Box sx={styles.user} onMouseLeave={() => setOpen(false)}>
       <Box onMouseEnter={() => setOpen(true)} sx={styles.userContainer}>
         <Box sx={styles.userInnerContainer}>
+          <Box>
+            <img
+              src={connectedLedger === 'Keplr' ? KeplrLogo : CosmostationLogo}
```

Comment:
I am thinking that having the two certain options covered and then defaulting is a bit safer, like:

`                src={
                  connectedLedger === 'Keplr' ? KeplrLogo :
                  connectedLedger === 'Cosmostation' ? CosmostationLogo :
                  WalletIcon
                }`

Also comparing to the predefined constants, such as CosmosNetworkConfig.KEPLR_LEDGER and CosmosNetworkConfig.COSMOSTATION_LEDGER

Reply:
Done